.github/workflows/test-m1.yml

```diff
@@ -19,6 +25,17 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Set CHANNEL (only for tagged pushes)
+        if: ${{ github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/') }}
+        run: |
+          # reference ends with an RC suffix
+          if [[ ${GITHUB_REF_NAME} = *-rc[0-9]* ]]; then
+            echo "CHANNEL=test" >> "$GITHUB_ENV"
+          fi
+      - name: Set Release CHANNEL (for release)
+        if: ${{ github.event_name == 'pull_request' && startsWith(github.base_ref, 'release') }}
```

Comment:
Same as above: why limit it to PR rather than to pushes to release branch?

Reply:
Done